app/pages/settings/email.jsx

```diff
@@ -251,6 +259,40 @@ class EmailSettingsPage extends React.Component {
               onChange={handleInputChange.bind(this.props.user)}
             />
```

Comment:
If you add `autocomplete='email'` to this input, that will help with accessibility too. See https://github.com/zooniverse/Panoptes-Front-End/issues/6944.

Reply:
This input should also be labelled with a HTML `<label for='inputId'>`, not a generic `<span>`.